spec/services/scm/create_managed_repository_service_spec.rb

```diff
@@ -134,4 +136,43 @@
       end
     end
   end
+
+  context 'with managed remote config' do
+    include_context 'with webmock'
```

Comment:
you could use metadata as well. https://www.relishapp.com/rspec/rspec-core/docs/hooks/filters#filter-`around(:example)`-hooks-using-arbitrary-metadata

maybe more obvious than a context? (maybe just gusto ...)